packages/common/sagas/entities/team.js

```diff
@@ -201,6 +190,116 @@ export function* createTeamKeyPairSaga({ payload: { team, publicKey } }) {
   }
 }
 
+/* 
+1. Get all public keys for the members
+1a. If user doesn't exist then create a keypair & masterpassword for that user
+2. Create a keypair clone from the team keypair per member
+3. Create secrets (encrypted keypairs) from the members keypairs
+4. Save all secrets to the server
+5. Add the members to the team
+
+*/
+function* encryptMemberTeamKey({ member, keypair }) {
+  const { id: userId, roles, publicKey } = member;
+  const secret = yield call(encryptSecret, {
+    item: convertKeyPairToItemEntity([keypair]),
+    publicKey,
+  });
+
+  return {
+    userId,
+    secret,
+    userRole: roles.includes(ROLE_ADMIN)
+      ? TEAM_ROLES.USER_ROLE_ADMIN
+      : TEAM_ROLES.USER_ROLE_MEMBER,
+  };
+}
+export function* addMemberToTeamListsBatchSaga({
+  payload: { teamId, members },
+}) {
+  try {
+    // const preparedMembers = yield call(prepareUsersForSharing, members);
```

Comment:
Is this commented code necessary? Maybe better to remove?

Reply:
![image](https://user-images.githubusercontent.com/167288/95859887-f9487500-0d67-11eb-8076-525631488d40.png)